snippets/isWeekday.md

```diff
@@ -0,0 +1,18 @@
+### isWeekday
+
+Results in a boolean representation of a specific date.
+
+Pass the specific date format string firstly. If not, use the current date by default.
+Use `new Date()` to let date format string be date object.
+Use `Date.getDay()` to check weekday then return a boolean.
+
+```js
+const isWeekday = (date = '') => {
```

Comment:
Wouldn't it be better if the default value was `new Date()` instead of `''`, so that you don't have to assign it below?

Reply:
@Chalarangelo. Yes, you'right.

This would be fine to let the `date` is `Date` object by default.